docs/source/rfcs/lts.rst

```diff
@@ -0,0 +1,358 @@
+==========================
+ RFC: Long Term Stability
+==========================
+
+This RFC contains proposals and clarifications regarding the
+maintenance and release processes of Avocado.
+
+We understand there are multiple teams currently depending on the
+stability of Avocado and we don't want their work to be disrupted by
+incompatibilities nor instabilities in new releases.
+
+This version is a minor update to previous versions of the same RFC
+(see `Changelog`_) which drove the release of Avocado 36.0 LTS.  The
+Avocado team has plans for a new LTS release in the near future, so
+please consider reading and providing feedback on the proposals here.
+
+TL;DR
```

Comment:
OK, now I found one occurence of that acronym... I hope that is not the case with our readers.